src/explore-education-statistics-admin/src/pages/publication/components/__tests__/LegacyReleasesTable.test.tsx

```diff
@@ -0,0 +1,283 @@
+import LegacyReleasesTable from '@admin/pages/publication/components/LegacyReleasesTable';
+import _legacyReleaseService from '@admin/services/legacyReleaseService';
+import { MyPublication } from '@admin/services/publicationService';
+import { render, screen, waitFor, within } from '@testing-library/react';
+import { MemoryRouter, Router } from 'react-router';
+import userEvent from '@testing-library/user-event';
+import { createMemoryHistory } from 'history';
+import React from 'react';
+
+jest.mock('@admin/services/legacyReleaseService');
+const legacyReleaseService = _legacyReleaseService as jest.Mocked<
+  typeof _legacyReleaseService
+>;
+
+describe('LegacyReleasesTable', () => {
+  const testPublication: MyPublication = {
+    id: 'publication-id-1',
+    title: 'Publication 1',
+    releases: [],
+    legacyReleases: [
+      {
+        description: 'Legacy release 3',
+        id: 'legacy-release-3',
+        order: 3,
+        publicationId: 'publication-id-1',
+        url: 'http://gov.uk/3',
+      },
+      {
+        description: 'Legacy release 2',
+        id: 'legacy-release-2',
+        order: 2,
+        publicationId: 'publication-id-1',
+        url: 'http://gov.uk/2',
+      },
+      {
+        description: 'Legacy release 1',
+        id: 'legacy-release-1',
+        order: 1,
+        publicationId: 'publication-id-1',
+        url: 'http://gov.uk/1',
+      },
+    ],
+    methodologies: [],
+    themeId: 'theme-1',
+    topicId: 'topic-1',
+    permissions: {
+      canAdoptMethodologies: true,
+      canCreateReleases: true,
+      canUpdatePublication: true,
+      canUpdatePublicationTitle: true,
+      canCreateMethodologies: true,
+      canManageExternalMethodology: true,
+    },
+  };
+  test('renders the legacy releases table correctly', () => {
+    render(<LegacyReleasesTable publication={testPublication} />);
+
+    const table = screen.getByRole('table');
+    const rows = within(table).getAllByRole('row');
+
+    expect(rows).toHaveLength(4);
+
+    const row1Cells = within(rows[0]).getAllByRole('columnheader');
+    expect(row1Cells[0]).toHaveTextContent('Order');
+    expect(row1Cells[1]).toHaveTextContent('Description');
+    expect(row1Cells[2]).toHaveTextContent('URL');
+    expect(row1Cells[3]).toHaveTextContent('Actions');
+
+    const row2Cells = within(rows[1]).getAllByRole('cell');
+    expect(row2Cells[0]).toHaveTextContent('3');
+    expect(row2Cells[1]).toHaveTextContent('Legacy release 3');
+    expect(row2Cells[2]).toHaveTextContent('http://gov.uk/3');
+    expect(within(row2Cells[3]).getByRole('button', { name: 'Edit release' }));
+    expect(
+      within(row2Cells[3]).getByRole('button', { name: 'Delete release' }),
+    );
```

Comment:
Ah interesting, it was actually because Duncan recently mentioned that it's superfluous. I've added toBeInTheDocument here to be consistent with other tests, but might be worth a discussion at a devs meeting.